zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -967,28 +979,23 @@ def process_avatars(avatar_list: List[ZerverFieldsT], avatar_dir: str,
         image_path = ('%s/%s.png' % (avatar_dir, avatar_hash))
         original_image_path = ('%s/%s.original' % (avatar_dir, avatar_hash))
 
-        # Fetch the avatars from the url
-        get_avatar(slack_avatar_url, image_path, original_image_path)
-        image_size = os.stat(image_path).st_size
+        avatar_upload_list.append([slack_avatar_url, image_path, original_image_path])
 
         avatar['path'] = image_path
         avatar['s3_path'] = image_path
-        avatar['size'] = image_size
```

Comment:
It looks like the size piece her got lost; is that intentional?

Reply:
@timabbott I removed it as I saw that it is not really required in the import process. See the function [import_uploads_local](https://github.com/zulip/zulip/blob/master/zerver/lib/export.py#L1415-L1456).